src/shared/util.js

```diff
@@ -8,6 +8,10 @@ export function isUndef (v: any): boolean %checks {
   return v === undefined || v === null
 }
 
+export function isNumNegZero (v: any): boolean %checks {
+  return !isNaN(v) && 1/Number(v) === -Infinity
```

Comment:
According [this eslint rule](https://eslint.org/docs/rules/no-compare-neg-zero), `Number(v) === -0` shall not be used. In addition, due to `Object.is` is not supported by IE. Therefore, `1/Number(v) === -Infinity` is used to determine whether `Number(v)` is `-0`

Reply:
In my testing, the `isNaN()` check is unnecessary, as is the `Number(v)` conversion. `return (1 / v) === -Infinity` should do it.